results/MetricRecorder.m

```diff
@@ -0,0 +1,168 @@
+classdef MetricRecorder
+	properties 
+		infoUtilLo;
+		infoUtilHi;
+		util;
+		power;
```

Comment:
is `power` explicit enough? Maybe `power_watt`

Reply:
good suggestion!